src/diffusers/models/attention_processor.py

```diff
@@ -2011,6 +2011,11 @@ def __call__(
         key = attn.head_to_batch_dim(key).contiguous()
         value = attn.head_to_batch_dim(value).contiguous()
 
+        if attn.norm_q is not None:
```

Comment:
Which test is this meant to fix?

Reply:
The CogVideoX tests. This was needed because we added QK norm in Attn2.0 and FusedAttn2.0.

However, after the new CogVideoX-5B PR, I think we can no longer support XFormers due to needing a custom attention processor. Maybe we can skip the test here because otherwise we'd need a custom XFormers processor variant for it?